decidim-initiatives/app/views/layouts/decidim/redesigned_initiative_creation.html.erb

```diff
@@ -0,0 +1,10 @@
+<%= append_javascript_pack_tag "decidim_initiatives" %>
+<%= append_stylesheet_pack_tag "decidim_initiatives", media: "all" %>
+<%= append_stylesheet_pack_tag "decidim_verifications", media: "all" %>
```

Comment:
This may cause some issues if verifications module is not enabled.

Reply:
We can remove this line, is actually not required